core/arch/arm/instr_create.h

```diff
@@ -115,6 +115,16 @@
 #define XINST_CREATE_load(dc, r, m)  INSTR_CREATE_ldr((dc), (r), (m))
 
 /**
+ * This platform-independent macro creates an instr_t which takes a 2-byte
```

Comment:
Hmm, I was thinking that expanding to 4 bytes would be the more frequently desired outcome.  On ARM ldrb always expands to 4 bytes, so it would match ARM.